JS/Lesson26/saffet/createPost.js

```diff
@@ -0,0 +1,46 @@
+const container = document.getElementById("container");
+const postArea = document.getElementById("postArea");
+const form = document.getElementById("postForm");
+const titleInput = document.getElementById("title");
+const bodyInput = document.getElementById("body");
+const saveButton = document.getElementById("saveButton");
+const successMessage = document.getElementById("successMessage");
+const urlParams = new URLSearchParams(window.location.search);
```

Comment:
You don't use post Id when sending the HTTP request so this and the next line are not needed :)